src/pages/instances/forms/UploadInstanceBackupFileForm.tsx

```diff
@@ -158,95 +136,96 @@ const UploadInstanceBtn: FC = () => {
-            close={handleCloseModal}
-            className="upload-instance-modal"
-            title="Upload instance"
-            buttonRow={
-              <>
-                <Button
-                  appearance="base"
-                  className="u-no-margin--bottom"
-                  type="button"
-                  onClick={handleCloseModal}
-                >
-                  Cancel
-                </Button>
-                <ActionButton
-                  appearance="positive"
-                  className="u-no-margin--bottom"
-                  loading={formik.isSubmitting || isUploading}
-                  disabled={
-                    !formik.isValid ||
-                    isProjectLoading ||
-                    !formik.values.instanceFile
-                  }
-                  onClick={() => void formik.submitForm()}
-                >
-                  Upload and create
-                </ActionButton>
-              </>
-            }
-          >
-            <Form
-              onSubmit={formik.handleSubmit}
-              className={uploadState ? "u-hide" : ""}
-            >
-              <Input
-                id="instance-file"
-                name="instanceFile"
-                type="file"
-                label="Instance backup file"
-                onChange={(e) => void changeFile(e)}
-              />
-              <Input
-                {...formik.getFieldProps("name")}
-                id="name"
-                type="text"
-                label="New instance name"
-                placeholder="Enter name"
-                error={formik.touched.name ? formik.errors.name : null}
-                disabled={!formik.values.instanceFile}
-              />
-              <StoragePoolSelector
-                project={project?.name || ""}
-                value={formik.values.pool}
-                setValue={(value) => void formik.setFieldValue("pool", value)}
-                selectProps={{
-                  id: "pool",
-                  label: "Root storage pool",
-                  disabled: isProjectLoading || !formik.values.instanceFile,
-                }}
-              />
-            </Form>
-            {uploadState && (
-              <>
-                <ProgressBar percentage={Math.floor(uploadState.percentage)} />
-                <p>
-                  {humanFileSize(uploadState.loaded)} loaded of{" "}
-                  {humanFileSize(uploadState.total ?? 0)}
-                </p>
-              </>
-            )}
-          </Modal>
-        </Portal>
-      )}
+      <ScrollableContainer
+        className={classnames({
+          "fixed-height": fixedFormHeight,
+          "u-hide": uploadState,
+        })}
+        dependencies={[]}
+        belowIds={["modal-footer"]}
+      >
+        <Form onSubmit={formik.handleSubmit}>
+          <Input
+            id="instance-file"
+            name="instanceFile"
+            type="file"
+            label="Instance backup file"
```

Comment:
Should this match the radio description, including the example file extensions? Might be better.

Reply:
Actually, what do you think about removing the label for this input? Since choosing the file type already relates to the file selection? For older LXD versions, we can show the label "LXD backup archive (.tar.gz)" since the file type selector will be hidden.

Reply:
Yes, I think that works well.